src/dashboard/src/main/tests/conftest.py

```diff
@@ -0,0 +1,40 @@
+import uuid
+
+from django.utils import timezone
+import pytest
+
+from main import models
+
+
+@pytest.fixture
+def transfer(db):
+    transfer = models.Transfer.objects.create(
+        uuid=uuid.uuid4(),
+        currentlocation=r"%transferDirectory%",
+        status=models.PACKAGE_STATUS_DONE,
+        completed_at=timezone.now(),
+    )
+
+    models.Job.objects.create(
+        sipuuid=transfer.pk, unittype="unitTransfer", createdtime=timezone.now()
+    )
+    models.File.objects.create(uuid=uuid.uuid4(), transfer=transfer)
+
+    return transfer
+
+
+@pytest.fixture
+def sip(db):
+    sip = models.SIP.objects.create(
+        uuid=uuid.uuid4(),
+        status=models.PACKAGE_STATUS_DONE,
+        completed_at=timezone.now(),
+    )
+
+    models.Job.objects.create(
+        sipuuid=sip.pk, unittype="unitSIP", createdtime=timezone.now()
+    )
+    models.File.objects.create(uuid=uuid.uuid4(), sip=sip)
+    models.Access.objects.create(sipuuid=sip.pk)
+
+    return sip
```

Comment:
It's great that you did this. I've been resisting a little because I wasn't sure what could be good reusable and combinable fixtures for every module, but you just came up with very good ones. I'm hoping we can have just a single conftest for all Archivematica components once we standardize the application layout and we stop messing with PYTHONPATH, lots of room there for cleaning things up. Thanks!

Reply:
My pleasure! And that sounds like a great goal - it'd be really nice to have a standard set of fixtures across the project.